telegram/ext/basepersistence.py

```diff
@@ -31,6 +31,33 @@
 from telegram.ext.utils.types import UD, CD, BD, ConversationDict, CDCData
 
 
+class PersistenceInput(NamedTuple):
+    """Convenience wrapper to group boolean input for :class:`BasePersistence`.
+
+    Args:
+        bot_data (:obj:`bool`, optional): Whether the setting should be applied for ``bot_data``.
```

Comment:
I dont think I like the sentence "Whether the setting". Why not call the thing by its name? "Whether this data (or data class?) will be saved".

Reply:
Because this class can be used for other settings, too. Specifically for the first part of #2524 …

Reply:
I still dont get why you have to give the arguments such unspecific strings instead of specific ones? Other settings can get their own specific strings after all?

Reply:
What I mean is with #2524, BPs init would be something like 

```python
def __init__(store_data: PersistenceInput = None, replace_bot: PersistenceInput = None):
```

where `replace_bot.bot_data` would tell you if `Bot` instances in `bot_data` should be replaced by `BP` before serialization or not. Now a doctsring `"Whether this data (or data class?) will be saved"` wouldn't make sense for `PersistenceInput.bot_data`.

Reply:
Alright, now I understand what you mean. I dont like it, but it's okay.